x-pack/metricbeat/module/aws/vpc/vpc_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package vpc
+
+import (
+	"os"
+
+	"github.com/elastic/beats/metricbeat/mb"
```

Comment:
Shouldn't that be together with other `beats` imports?

Reply:
The other two `beats` import are for registering input module and metricset 😄 So I left them separate with a comment.